spec/factories/course_enrichments.rb

```diff
@@ -99,16 +99,22 @@
     last_published_timestamp_utc { 5.days.ago }
   end
 
-  trait :with_invalid_content do
+  trait :with_invalid_content_exceed_word_count_fields do
```

Comment:
Can we create this in the spec instead as a before block?

Reply:
didn't use the before block after all, just inlined when creating it as well as changing the underlying test mark up